fvm/errors.go

```diff
@@ -29,6 +30,8 @@ const (
 
 	errCodeEventLimitExceededError = 20
 
+	errCodeEncodingUnsupportedValue = 30
```

Comment:
Didn't know what to use here

Reply:
any value is a good value for this version of fvm